aptos-move/framework/aptos-framework/sources/aptos_coin.move

```diff
@@ -34,18 +34,19 @@ module aptos_framework::aptos_coin {
     }
 
     /// Can only called during genesis to initialize the Aptos coin.
-    public(friend) fun initialize(aptos_framework: &signer): (MintCapability<AptosCoin>, BurnCapability<AptosCoin>) {
+    public(friend) fun initialize(aptos_framework: &signer): (BurnCapability<AptosCoin>, MintCapability<AptosCoin>) {
         system_addresses::assert_aptos_framework(aptos_framework);
 
-        let (mint_cap, burn_cap) = coin::initialize<AptosCoin>(
+        let (burn_cap, freeze_cap, mint_cap) = coin::initialize<AptosCoin>(
             aptos_framework,
             string::utf8(b"Aptos Coin"),
             string::utf8(b"APT"),
             8, /* decimals */
             false, /* monitor_supply */
         );
 
-        (mint_cap, burn_cap)
+        coin::destroy_freeze_cap(freeze_cap);
+        (burn_cap, mint_cap)
```

Comment:
Note, anyone else who uses the test signature will now be confused

Reply:
confused ... maybe but it is statically typed :)